rfd/9999-oidc-idp-for-aws-rds-dbs-discovery.md

```diff
@@ -0,0 +1,495 @@
+---
+authors: Marco Dinis (marco.dinis@goteleport.com)
+state: draft
+---
+
+# RFD 999 - OIDC IdP for AWS RDS DBs discovery
+
+## Required Approvals
+
+* Engineering: @ryanclark && @r0mant
+* Product: @xinding33 || @klizhentas
+* Security: @reedloden
+
+## What
+
+Discover AWS RDS Databases without manual configuration.
+
+#### Goals
+
+Easily discover AWS RDS Databases of an account.
```

Comment:
I was targeting something more specific to avoid the "where are we going to use this"

Let me rephrase the goal and RFD title to "Integrate with AWS using OIDC"